README.md

```diff
@@ -1,24 +1,51 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Math_magician
 
-Things you may want to cover:
+> The Blog app will be a classic example of a blog website. I will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- RoR
 
-* Configuration
+## Live Demo (Not yet available)
 
-* Database creation
+You can check the live demo :
+### Prerequisites
 
-* Database initialization
+- Have completed all the previous learning of modules.
+  `(HTML, CSS and JS, Database, Ruby)`
 
-* How to run the test suite
+## Set up
 
-* Services (job queues, cache servers, search engines, etc.)
+- `git clone git@github.com:christianbiring1/Blog_App.git`
+- `rails new Blog_App --database=postgresql`
+- `cd Blog_App`
+- `rails s`
+- `localhost.300`
 
-* Deployment instructions
+## Author
 
-* ...
+👤 **Christian biringanine**
+
+- GitHub: [@christianbiring1](https://github.com/christianbiring1)
+- Twitter: [@christianbirin4](https://twitter.com/christianbirin4)
+- LinkedIn: [Christian Biringanine](https://linkedin.com/in/christian-biringanine-1833011a5/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone who will take a glance to this project
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
[OPTIONAL] This MIT link is broken. I recommend fixing this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.